### ISSUE TITLE
Can't use a paper-radio-button imperatively

### ISSUE THREAD

**Issue by cedric-marcone** [PolymerElements/paper-radio-button#26]
Hello !

Trying to instantiate a `paper-radio-button` from code it fails because the label stays hidden.

``` js
var b = document.createElement('paper-radio-button')
var l = document.createTextNode('my label')
Polymer.dom(b).appendChild(l)
```

As the label is no longer a property, the only possible solutions I can see would be to :

**Add a MutationObserver**
-> Seems heavy

**Handle the hiding logic in the `attached` event handler**
-> It would not handle mutation scenarios but surely would work for the general use-cases needing imperative radio buttons generation.

**Comment by gertcuykens** [comment]
What is the current work around please?

**Comment by notwaldorf** [comment]
We are waiting on a fix in the Polymer core library, that will allow us to observe changes in `<content>`.

**Comment by gertcuykens** [comment]
Also when you create it this way the paper-radio-group is not working and can select both buttons. Don't know if this is a separate issue?

![image](https://cloud.githubusercontent.com/assets/863731/8363925/bacb4a64-1b82-11e5-8dd9-04a6e84dc26d.png)

**Comment by dnegstad** [comment]
Using the attached callback instead of ready seems to allow paper-radio-button (and paper-checkbox) to work both when the element starts as part of the page as well as when it's added programmatically.

**Comment by notwaldorf** [comment]
@dnegstad Yeah, that works. I think while we're waiting on the real fix, I'll make that change. Thanks!
